vec-rdf/vec-rdf-common/src/main/java/com/foursoft/harness/vec/rdf/common/meta/xmi/VersionLookupModelProvider.java

```diff
@@ -0,0 +1,63 @@
+/*-
+ * ========================LICENSE_START=================================
+ * VEC RDF Common
+ * %%
+ * Copyright (C) 2024 4Soft GmbH
+ * %%
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ * =========================LICENSE_END==================================
+ */
+package com.foursoft.harness.vec.rdf.common.meta.xmi;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class VersionLookupModelProvider implements VecModelProvider {
+    private final Map<Package, VecModelProvider> providerMap = new HashMap<>();
+    private static final Package v113Package = com.foursoft.harness.vec.v113.VecContent.class.getPackage();
```

Comment:
This class should be written differently so the caller provides the information that needs both modules (rdf and vec).